services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -721,6 +721,28 @@ CREATE OR REPLACE PROCEDURE
   END;
 $$
 
+CREATE OR REPLACE PROCEDURE
+  add_enum_microsoftteams_to_type_in_project_notification()
+
+  BEGIN
+    DECLARE column_type_project_notification_type varchar(50);
+
+    SELECT COLUMN_TYPE INTO column_type_project_notification_type
+    FROM INFORMATION_SCHEMA.COLUMNS
+    WHERE
+      table_name = 'project_notification'
+      AND table_schema = 'infrastructure'
+      AND column_name = 'type';
+
+    IF (
+      column_type_project_notification_type = "enum('slack')"
```

Comment:
I think this should be `"enum('slack','rocketchat')"` because at this point the DB already knows about the type `slack` and `rocketchat`